.github/workflows/pre-submit.yml

```diff
@@ -20,7 +20,16 @@ jobs:
           set -euo pipefail
 
           go mod vendor
-          # Build
+          
+          # Build cli
+          cd cli
```

Comment:
simplify to remove the `cd .../` with `go build -mod=vendor ./cli` and `go build -mod=vendor ./service`?

Reply:
I tried that but it failed `go: build output "cli" already exists and is a directory`... any way around that?

Reply:
`-o verifier`? it's weird i can't find any docs on that... i'm pretty sure the behavior for `./cmd/XYZ` is that it writes to XYZ. But maybe not.

Reply:
`-o` did the trick. Thanks